apps/meteor/ee/client/hooks/useHasLicense.ts

```diff
@@ -1,13 +1,17 @@
 import { useState, useEffect } from 'react';
 
 import { hasLicense } from '../../app/license/client';
-import { BundleFeature } from '../../app/license/server/bundles';
 
-export const useHasLicense = (licenseName: BundleFeature): 'loading' | boolean => {
+export const useHasLicense = (licenseName: string): 'loading' | boolean => {
```

Comment:
why?

Reply:
My bad, I think I re-add this as a String when I was doing a merging. I will add it back